lib/imagemounter/index.js

```diff
@@ -0,0 +1,119 @@
+import { readFile, createReadStream } from 'fs';
```

Comment:
please use async readFile helper from the corresponding `@appium/support`'s fs module